### ISSUE TITLE
Set metadata to the correct supported platforms?

### ISSUE THREAD

**Issue by feniix** [nibalizer/puppet-module-zfs#4]
Hello, would you mind changing the metadata of your module to correctly reflect the supported platforms?

Thanks in advance

**Comment by nibalizer** [comment]
No. this code will melt your computer. Do not use. You have my blessing to fork and develop on your own though! I'll hapily point to yours as the upstream.

**Comment by feniix** [comment]
hahah, that is a very honest response!

thanks
